services/core/shared/core/delegates.js

```diff
@@ -34,7 +34,7 @@ const delegateStatus = {
 	STANDBY: 'standby',
 	BANNED: 'banned',
 	PUNISHED: 'punished',
-	NON_ELIGIBLE: 'non-eligible',
+	NON_ELIGIBLE: 'ineligible',
```

Comment:
Please also rename the constant identifier at the necessary places.
```suggestion
	INELIGIBLE: 'ineligible',
```